src/pages/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,72 @@
+import {FC} from "react";
+import {useNavigate} from "react-router-dom";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+import VisibilityOffIcon from "@mui/icons-material/VisibilityOff";
+import {Button} from "@containers/Button";
+import {Input} from "@containers/Input";
+import {HeaderAuth} from "@view/AuthHeader/HeaderAuth";
+import {PageContainer} from "@view/PageContainer/PageContainer";
+import {ElemStyle, ContentStyle, PasswordIconStyle} from "@view/MuiPagesStyles";
+import {Pages} from "@/models/Pages";
+import {useBooleanState} from "@/hooks/useBooleanState";
+
+export const SignIn: FC = () => {
+  const [pswrdVisibility, setVisibilityTrue, setVisibilityFalse] =
```

Comment:
```suggestion
  const [passwordVisibility, setPassowordVisibilityTrue, setPassowordVisibilityFlase] =
```

не сокращай названия